pkg/pillar/cmd/nim/nim.go

```diff
@@ -803,6 +799,12 @@ func (n *nim) handleZedAgentStatusImpl(_ string, statusArg interface{}) {
 // Otherwise the ShaFile and Shavalue is used to write the sha for the new file to avoid
 // re-application of the same config.
 func (n *nim) ingestDevicePortConfig() {
+	// Skip these legacy DPC json files if there is bootstrap config.
+	if _, err := os.Stat(types.BootstrapConfFileName); err == nil {
+		n.Log.Notice("Not ingesting DPC jsons from config partition: " +
```

Comment:
This message makes one believe that there were some to ingest, which isn't necessarily the case.
How about moving down this block and prefix it with len(locations) != 0? That way the log message will indicate that something was ignored.

Reply:
Makes sense, done